### ISSUE TITLE
Non featured image in posts section produces a warning

### ISSUE THREAD

**Issue by Letme** [wangchucheng/hugo-eureka#13]
I have a simple example and not sure if it is a problem or not (or expected behavior). In `content/../something.md` you want to place an image with name `xyz.jpg` using standard markdown syntax. When you run Hugo, you receive error:

```
ERROR 2020/11/27 00:50:59 render of "section" failed: execute of template failed: template: products/list.html:3:4: executing "products/list.html" at <partial "head.html" .>: error calling partial: execute of template failed: template: partials/head.html:80:3: executing "partials/head.html" at <partial "schema.html" .>: error calling partial: execute of template failed: template: partials/schema.html:23:3: executing "partials/schema.html" at <partial "schema_node.html" .>: error calling partial: execute of template failed: template: partials/schema_node.html:15:24: executing "partials/schema_node.html" at <partial "get_image.html" (dict "context" . "url" .Params.featuredImage "keyword" "*featured*")>: error calling partial: "/private/path/themes/eureka/layouts/partials/get_image.html:2:76": execute of template failed: template: partials/get_image.html:2:76: executing "partials/get_image.html" at <.keyword>: nil pointer evaluating resource.Resource.Permalink
ERROR 2020/11/27 00:50:59 render of "term" failed: execute of template failed: template: _default/list.html:3:4: executing "_default/list.html" at <partial "head.html" .>: error calling partial: execute of template failed: template: partials/head.html:80:3: executing "partials/head.html" at <partial "schema.html" .>: error calling partial: execute of template failed: template: partials/schema.html:23:3: executing "partials/schema.html" at <partial "schema_node.html" .>: error calling partial: execute of template failed: template: partials/schema_node.html:15:24: executing "partials/schema_node.html" at <partial "get_image.html" (dict "context" . "url" .Params.featuredImage "keyword" "*featured*")>: error calling partial: "/private/path/themes/eureka/layouts/partials/get_image.html:2:76": execute of template failed: template: partials/get_image.html:2:76: executing "partials/get_image.html" at <.keyword>: nil pointer evaluating resource.Resource.Permalink
Total in 209 ms
Error: Error building site: failed to render pages: render of "section" failed: "/private/path/layouts/products/list.html:2:76": execute of template failed: template: products/list.html:2:3: executing "main" at <partial "doc_layout.html" .>: error calling partial: execute of template failed: template: partials/doc_layout.html:18:31: executing "partials/doc_layout.html" at <partial "get_image.html" (dict "context" . "url" .Params.featuredImage "keyword" "*featured*")>: error calling partial: "/private/path/themes/eureka/layouts/partials/get_image.html:2:76": execute of template failed: template: partials/get_image.html:2:76: executing "partials/get_image.html" at <.keyword>: nil pointer evaluating resource.Resource.Permalink
```

This error goes away, if I place an image with `featured` in name in the same directory as markdown then error goes away.

This is a bug for me, because I do not want a featured image, but I do want an image in the markdown. I would assume a simple fix inside the `get_image.html`, but I lack knowledge of hugo or go to solve it quickly. Anyone can help with a quickfix?

**Comment by wangchucheng** [comment]
@Letme I will fix this bug this evening (GMT+8). Thank you for your feedback!

**Comment by wangchucheng** [comment]
@Letme  This bug has been fixed in d998e9f.

Now you can use any name you want in the content folder. You can checkout `master` to preview it or wait for the next release which may be released in hours.

Thank you for your feedback to Eureka!

**Comment by Letme** [comment]
I can confirm the fix works on my site as well. Thanks for the quick fix and have a nice night.
